core-kotlin-modules/core-kotlin-collections-list-2/src/test/kotlin/removeNullAndEmptyValues/RemoveNullAndEmptyValuesUnitTest.kt

```diff
@@ -0,0 +1,44 @@
+package removeNullAndEmptyValues
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+class RemoveNullAndEmptyValuesUnitTest {
+
+    @Test
+    fun `remove null and empty values from list via list iteration`() {
+        val listWithNullsAndEmpty: MutableList<String?> = mutableListOf("A", null, "", "C", null, "E")
+        val listWithoutNullsAndEmpty = removeValuesViaIteration(listWithNullsAndEmpty)
+        assertEquals(listOf("A", "C", "E"), listWithoutNullsAndEmpty)
+    }
+
+    @Test
+    fun `remove null and empty values from list using filterNotNull and filterNot methods`(){
```

Comment:
Just some spacing cleanup here:
```suggestion
    fun `remove null and empty values from list using filterNotNull and filterNot methods`() {
```